addons/mass_mailing/models/utm.py

```diff
@@ -28,14 +30,18 @@ class UtmCampaign(models.Model):
     replied_ratio = fields.Integer(compute="_compute_statistics", string='Replied Ratio')
     bounced_ratio = fields.Integer(compute="_compute_statistics", string='Bounced Ratio')
 
+    def _compute_mailings_count(self):
+        for campaign in self:
+            campaign.mailings_count = len(campaign.mailing_ids)
+
     @api.depends('mailing_items', 'mailing_clicked')
     def _compute_items_total(self):
+        super(UtmCampaign, self)._compute_items_total
```

Comment:
Missing "()"

Leads me to wonder if you have tested mailing + social and check that stats are correctly summed.
Might be worth to check...